spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -233,6 +233,21 @@ configuration.classes=org.springframework.cloud.stream.app.file.sink.FileSinkPro
 configuration-properties.names=server.port
 ```
 
+=== Global application properties (for streams)
+
+In addition to configuration via DSL, Spring Cloud Data Flow provides a mechanism for setting specific properties to all the streaming applications that are launched by it.
+This can be done by adding properties prefixed with `spring.cloud.dataflow.applicationProperties.stream` when starting the server.
+When doing so, the server will pass all the properties, without the prefix, to the instances it launches.
+
+For example, all the launched applications can be reconfigured to use a different Kafka broker by the default by launching the configuration server with the following options:
```

Comment:
@mbogoevici Given these common application properties are for the dataflow server and launching the `config` server with the new properties would cause the refresh on the dataflow server configuration only. Is that right? In that case, how can we expect the deployed apps to be refreshed with these properties. what am I missing?